@kiva/kv-components/src/vue/KvCarousel.vue

```diff
@@ -35,8 +35,12 @@
 			v-if="slideIndicatorCount > 1 && !isDotted"
 			class="kv-carousel__controls tw-flex
 			tw-justify-between md:tw-justify-center tw-items-center
-			tw-mt-4 tw-w-full"
-			:class="{ 'lg:tw-hidden': asideControls }"
+			tw-w-full"
+			:class="{
+				'lg:tw-hidden': asideControls,
+				'tw-mb-4 tw-mt-4': !controlsTopRight,
```

Comment:
Do we need this new/additional bottom margin `tw-mb-4` applied globally? Seems like that will bump up the height a good bit on all carousels...